### PR TITLE
MacPorts_to_Jhbuild

### PR DESCRIPTION
**Pull request by W0naN0w**
**Changing from Macports to Jhbuild for libapr and libsvn.**
Now using static libraries into nmap-update for libapr and libsvn (and libintl - it is a dependency installed by Jhbuild).

### CODE REVIEW

File: macosx/Makefile [hunk 1]
```diff
@@ -33,7 +34,7 @@ JHBUILD_PREFIX=$(HOME)/gtk/inst
 export JHBUILD_PREFIX
 # MACPORTS_PREFIX is still here because of nmap-update, which needs
 # libsvn and libapr, which jhbuild doesn't support yet.
-MACPORTS_PREFIX=$(HOME)/macports-10.8
+# MACPORTS_PREFIX=$(HOME)/macports-10.8
```
Comment (dmiller-nmap) [inline macosx/Makefile]: We can delete this line instead of commenting it out.

File: macosx/Makefile [hunk 2]
```diff
@@ -70,20 +71,33 @@ LDFLAGS += $(UNIVERSAL_LDFLAGS)
...
-path-check:
-	if (echo $(PATH) | grep -F "$(MACPORTS_PREFIX)"); then false; else echo "Path is OK"; fi
-
 tool-checks: have-$(CC) have-jhbuild have-gtk-mac-bundler
 
 have-%:
 	which $*
 
-$(IMAGE_STAGING_DIR)/$(PKG_NAME): check-nmap check-ncat check-ndiff check-zenmap check-nping check-nmap-update COPYING.formatted
+$(IMAGE_STAGING_DIR)/$(PKG_NAME): check-nmap-update COPYING.formatted # check-nmap check-ncat check-ndiff check-zenmap check-nping
```
Comment (dmiller-nmap) [inline macosx/Makefile]: Why are the rest of these commented out? Guessing that you wanted to test more quickly, but they should be restored in the final merge.

Comment (W0naN0w) [inline macosx/Makefile]: Oh I thought I modified this, but yes it was to test more quickly!

File: nmap-update/Makefile.in [hunk 3]
```diff
@@ -10,7 +10,7 @@ CC = @CC@
 CFLAGS = @CFLAGS@
 CPPFLAGS = @CPPFLAGS@ $(DEFS) -I$(NBASEDIR)
 LDFLAGS = @LDFLAGS@
-LIBS = @LIBS@ $(NBASELIB)
+LIBS = @LIBS@ $(NBASELIB) -lsqlite3 -lz -lsasl2 -liconv -framework CoreFoundation -framework Security
```
Comment (dmiller-nmap) [inline nmap-update/Makefile.in]: This Makefile is intended for use on all platforms, so platform-specific stuff like this should either be handled in the configure script or in the `macosx/Makefile`. Check whether it's needed for dynamic linking build by just doing a `./configure && make nmap-update` from the top-level source dir on OS X. If not, then it's specific to our packaging process and should go into the `macosx/Makefile`.

File: nmap-update/Makefile.in [hunk 4]
```diff
@@ -59,6 +59,6 @@ Makefile: Makefile.in
 
 makefile.dep:
 	$(CC) -MM $(CPPFLAGS) $(C_FILES) > $@
-include makefile.dep
+	include makefile.dep
```
Comment (dmiller-nmap) [inline nmap-update/Makefile.in]: I don't think this is the right fix for the errors you said you were getting. Can you be more specific about what the errors are and when they show up?

Comment (W0naN0w) [inline nmap-update/Makefile.in]: Of course! Well it is maybe not errors but Makefiles tells me "Makefile:xxx(line number): makefile.dep: No such file or directory" several times (for each Makefile I suppose). Here is a paste of my stdout : http://pastebin.com/RVJHdz6P

**Comment by W0naN0w** [comment]
Redirected to #392

**Pull request closed without merging**